File: java/src/main/java/be/steinsomers/bron_kerbosch/BronKerbosch3.java
package be.steinsomers.bron_kerbosch;

import java.util.stream.Stream;

public final class BronKerbosch3  implements BronKerboschAlgorithm {
    @Override
    public Stream<int[]> explore(UndirectedGraph graph) {
        return BronKerboschOrder.explore(graph, PivotChoice.Arbitrary);
    }
}

File: kotlin/src/main/be/steinsomers/bron_kerbosch/BronKerbosch3.kt
package be.steinsomers.bron_kerbosch

class BronKerbosch3 : BronKerboschAlgorithm {
    override fun explore(graph: UndirectedGraph, cliqueConsumer: (IntArray) -> Unit) {
        BronKerboschDegeneracy.explore(graph, cliqueConsumer, PivotChoice.Arbitrary)
    }
}
